uiowa/uiowa
# [CLAS] redirects

Resolves https://github.com/uiowa/uiowa/issues/7761. 

# How to test

```
ddev blt ds --site=clas.uiowa.edu 
```
1. Test a few redirects to see if they were set.
